apps/ios/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "FluentTester-ios",
   "version": "0.0.1",
   "private": true,
-  "main": "src/index.tsx",
+  "main": "index.tsx",
   "module": "src/index.tsx",
```

Comment:
can presumably change this too since index.tsx just references src/index.tsx

Reply:
This may be an unintentional change, I'll make sure it's necessary or not.

Reply:
I actually think it's the right idea but we probably need to make it index.js instead of .tsx.